src/scorecard.js

```diff
@@ -0,0 +1,73 @@
+var Scorecard = function(content) {
+  this.frameRecord = [];
+  this.totals = [];
+  this.DEFAULT_FRAMES = 10;
+  this.currentTurn = 0;
+  this.maxTurns = 20;
+  for (var i = 0; i < this.DEFAULT_FRAMES; i++) {
+    this.frameRecord.push(new content);
+  };
+};
+
+Scorecard.prototype.updateTotal = function() {
+  if (this.frameRecord[this.getCurrentFrame()].strike === true || this.frameRecord[this.getCurrentFrame()].spare === true) {
```

Comment:
DRY!!!!

How many times is this repeated? `this.frameRecord[this.getCurrentFrame()]` throughout this object?

`getCurrentFrame` should return the current frame, as the method name implies, not the current frame number.  This single change in your code would improve the readability of this object at least tenfold.